website/src/css/custom.css

```diff
@@ -62,3 +62,14 @@
 .anchor {
   scroll-margin-top: 50pt;
 }
+
+@media (max-width: 996px) {
+  .navbarSearchContainer_node_modules-\@docusaurus-theme-classic-lib-theme-Navbar-Search-styles-module  {
+    position: relative !important;
+    right: 0 !important;
+  }
+
+  .navbar__items--right > .navbar__item:first-child {
```

Comment:
This isn't ideal either, but I don't see a clean way to inject custom classes/ID to those navbar elements via docusaurus.